src/core/PassphraseGenerator.cpp

```diff
@@ -98,7 +109,23 @@ QString PassphraseGenerator::generatePassphrase() const
     QStringList words;
     for (int i = 0; i < m_wordCount; ++i) {
         int wordIndex = randomGen()->randomUInt(static_cast<quint32>(m_wordlist.length()));
-        words.append(m_wordlist.at(wordIndex));
+        tmpWord = m_wordlist.at(wordIndex);
+
+        //convert case
+        switch (m_wordCase) {
+        case 0 : // lower case
+            tmpWord = tmpWord.toLower();
+            break;
+        case 1 : // UPPER CASE
+            tmpWord = tmpWord.toUpper();
+            break;
+        case 2 : // Title Case
+            tmpWord = tmpWord.replace(0,1,tmpWord.left(1).toUpper());
+            break;
+        default :
```

Comment:
The default should mimic LOWERCASE and do toLower

Reply:
changes made